modules/electromagnetics/test/include/functions/JinSlabCoeffFunc.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "Function.h"
+
+/**
+ *    Function for field coefficient in slab reflection benchmark case.
+ */
+class JinSlabCoeffFunc : public Function
+{
+public:
+  static InputParameters validParams();
+
+  JinSlabCoeffFunc(const InputParameters & parameters);
+
+  using Function::value;
+  virtual Real value(Real t, const Point & p) const override;
+
+protected:
+  Real _k;
+
+  Real _theta;
+
+  Real _length;
+
+  Real _coef;
+
+  MooseEnum _component;
```

Comment:
const and comments

Reply:
Even in a test function? I don't mind it, just didn't consider it because it wouldn't show up in doco.

Reply:
well comments are for maintainers here
Alex wasnt too keen on it for NS as well.

Reply:
Don't mind doing it for clarity and completeness.